### PR TITLE
SPM, Pods, and Tests fixes

### PR DESCRIPTION
**Pull request by ypopovych**

### CODE REVIEW

**Comment by dileping** [comment]
@ypopovych  Build fails in CI. Could you do something about it?

**Comment by ypopovych** [comment]
Fixed Xcode crashes on CI. Linux builds must work after the release of Boilerplate.

File: Regex/NSRegularExpression3.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//
+//  NSRegularExpression3.swift
+//  Regex
+//
+//  Created by Yegor Popovych on 3/31/16.
+//  Copyright © 2016 Crossroad Labs, LTD. All rights reserved.
```
Comment (dileping) [inline Regex/NSRegularExpression3.swift]: Copyright

File: CrossroadRegex.podspec [hunk 2]
```diff
@@ -14,6 +14,8 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
 
   s.source_files = 'Regex/*.swift'
+  
+  s.dependency 'CrossroadBoilerplate', '~> 0.1'
```
Comment (dileping) [inline CrossroadRegex.podspec]: Correct dependency

**Pull request merged**